### DISCUSSION TITLE
Logs in Zigbee2MQTT being updated every 5 seconds by one specific device.

### DISCUSSION THREAD

**Discussion by Mi-Nigle** [Koenkk/zigbee2mqtt#18546] [Devices]
I have a handful of devices connected to Zigbee2MQTT. Four of these devices are Hive plugs, and one of these hive plugs is updating the logs every 5 seconds, and I don't know why.

![image](https://github.com/Koenkk/zigbee2mqtt/assets/66018667/73ee6e1a-b3f2-44a4-98c2-2219bf78d28a)

If I turn it off and on again, the updates stop until I use it, then they never stop.

I've compared the settings within Zigbee2MQTT to the other plug devices and they're identical as far as I can see.

It's not causing any problems as far as I'm aware, but I'd still like to know why it differs from the other plugs. Does anyone have any ideas?

Thank you.

**Comment by lux73** [comment]
I would assume this Device works with a static polling all 5 seconds

This means your Device sends all 5 Seconds Status updates even there is no change

**Comment by Mi-Nigle** [comment]
Thank you for your reply!
